client/src/components/UI/Button/Button.css

```diff
@@ -0,0 +1,7 @@
+@media(max-width : 500px){
+    .btn{
+       width: 90px !important;
+       height: 40px !important;
```

Comment:
You should avoid important in css, maybe you want to search more how we can handle responsive in bootstrap